### ISSUE TITLE
SSH Key Missing for link upload

### ISSUE THREAD

**Issue by mbuhrley** [TechnoSwiss/broadcast_pi#1]
I am able to ssh from the pi into our site using the passphrase stored in the SSH_RSA_KEY_PASS file however I keep getting the error "SSH Key Missing for link upload" when I try to run the broadcast.py script with our options. 

Also, I may be missing something but I couldn't find a clear indication of what the -k (url-key) option is referring to. The help text "The 4-digit code at the end of the URL" has me scratching my head as I am not seeing a particular url with a 4 digit code at the end, do I just grab the last 4 of the youtube live stream url?

**Comment by TechnoSwiss** [comment]
I need to add some more argument checking to make sure everything that's needed is included on the command line. Have you used -D to specify the home directory that your SSH keyfile lives in? Also, is your ssh key in the standard .ssh/id_rsa location?

Yup I was just looking at that when the other day thinking I should probably explain that better. The -k (url-key) option is really just to obscure the URL if you're doing this for multiple wards, So instead of the URLs just being warda.html, wardb.html they would be ward_abcd.html or ward_1234.html, the -k option gets appended to the end. When I was selecting 4 digit codes to use, I just used a password generator to create alpha_numeric codes, and I excluded i, l, 0 & o from the allowed list so there weren't any codes created that could cause confusion over what the characters were.

**Comment by mbuhrley** [comment]
Yes to both the -D argument and the key location but no luck getting past that error.

ok, that makes sense now that you put it that way regarding the -k option.
How about the Twilio and phone arguments, necessary? 

The other setup was pretty straight forward, setting up the youtube data api and google authentication.

**Comment by TechnoSwiss** [comment]
I was miss-reading which error code you were getting "SSH Key Missing for link upload" means it can't find the ssh.pass file, which should be a plane text file with a single line, which is the password for your SSH key, it should be in the same directory as the scripts.

I'm thinking you might run into an issue with the upload as well since I have hard coded the path for the uploaded file to public_html/broadcast/ so you might either want to verify that those exist, or change the script to match your directory structure. I should make that a parameter.

The Twilio and phone arguments are just for sending text messages about error messages, if they're not included it's ignored, and isn't required.
